controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1287,6 +1291,14 @@ int cli_bml::bml_wfca_agent_caller(int numOfArgs)
     return -1;
 }
 
+int cli_bml::bml_trigger_topology_discovery_caller(int numOfArgs)
+{
+    if (numOfArgs == 1) {
+        return topology_discovery(args.stringArgs[0]);
```

Comment:
no verification for user argument, potential security breach. Need to convert to `sMacAddr` and if fails return error. Then pass the struct instead of const char *.

Reply:
see previous comment